gradle/libs.versions.toml

```diff
@@ -89,3 +95,9 @@ okhttp = { module = "com.squareup.okhttp3:okhttp", version.ref = "okhttp" }
 okhttp-mockwebserver = { module = "com.squareup.okhttp3:mockwebserver", version.ref = "okhttp" }
 
 tree-printer = "hu.webarticum:tree-printer:3.0.0"
+core-ktx = { group = "androidx.core", name = "core-ktx", version.ref = "core-ktx" }
+junit-junit = { group = "junit", name = "junit", version.ref = "junit" }
+androidx-test-ext-junit115 = { group = "androidx.test.ext", name = "junit", version.ref = "androidx-test-ext-junit" }
+espresso-core = { group = "androidx.test.espresso", name = "espresso-core", version.ref = "espresso-core" }
```

Comment:
Not sure this is necessary for anything

Reply:
I think AS added this in automatically, I sure didn't 😳

Reply:
Oh I think it automatically created an Espresso test when I created the module, so can delete all of this.